### PR TITLE
feat(project-creation): Create a field on team model for tracking

### PR DESCRIPTION
**Pull request by schew2381**
Closed in favor of https://github.com/getsentry/getsentry/pull/10645

### CODE REVIEW

**Review by cathteng — COMMENTED** [review]
are you planning to track this through redash? iirc the flag field in redash is the sum of all the flags that are active for the project or something like that

**Comment by schew2381** [comment]
> are you planning to track this through redash? iirc the flag field in redash is the sum of all the flags that are active for the project or something like that

I'm not sure I understand, can I use a WHERE clause to filter for teams with `through_project_creation`?

**Comment by cathteng** [comment]
oh sorry i thought this was a flag. ignore me!

File: src/sentry/models/team.py [hunk 1]
```diff
@@ -160,6 +160,7 @@ class Team(Model, SnowflakeIdMixin):
     idp_provisioned = models.BooleanField(default=False)
     date_added = models.DateTimeField(default=timezone.now, null=True)
     org_role = models.CharField(max_length=32, null=True)
+    through_project_creation = models.BooleanField(default=False)
```
Comment (cathteng) [inline src/sentry/models/team.py:163]: is this a permanent field? or will we drop it after the experiment is done?

Comment (schew2381) [reply src/sentry/models/team.py:163]: hmm I presume we would drop if after the experiment, although if it ends up successful and we want to reimplement it properly we could re-use this field

**Review by markstory — COMMENTED** [review]
Will this new field only be used for tracking flows, or will it be used in application logic that is coming in the future? If you only want to track we could use logging or sentry events to record occurrences.

**Comment by wedamija** [comment]
> Will this new field only be used for tracking flows, or will it be used in application logic that is coming in the future? If you only want to track we could use logging or sentry events to record occurrences.

Yeah this field feels a little weird to me, would definitely like to understand the use case for it in more detail

**Comment by schew2381** [comment]
> Will this new field only be used for tracking flows, or will it be used in application logic that is coming in the future? If you only want to track we could use logging or sentry events to record occurrences.

This will only be used for tracking flows so I realize this is a bad change. I'm switching to use the audit log to track teams that are created through the new project creation flow instead

**Pull request closed without merging**